### PR TITLE
can now send as sms only

### PR DESCRIPTION
**Pull request by andreasastrom**

### CODE REVIEW

File: GetAccept-v2/app.js [hunk 1]
```diff
@@ -1284,10 +1287,34 @@ lbs.apploader.register('GetAccept-v2', function () {
 
         viewModel.createEmail = packEmailData;
 
+        function activateSmsSending() {
```
Comment (j-tegen) [inline GetAccept-v2/app.js]: Would be better as a `ko.computed`?

File: GetAccept-v2/app.js [hunk 2]
```diff
@@ -1284,10 +1287,34 @@ lbs.apploader.register('GetAccept-v2', function () {
...
+            var recipientWithoutEmail = []; 
+            if (viewModel.sendSMS()) {
+                return;
+            }
+            recipientWithoutEmail = viewModel.recipientsList().filter(function(recipeint) {
```
Comment (j-tegen) [inline GetAccept-v2/app.js]: No need to pre-declare `recipientWithoutEmail` before this row.

File: GetAccept-v2/app.js [hunk 3]
```diff
@@ -689,7 +690,7 @@ lbs.apploader.register('GetAccept-v2', function () {
             originalPersonList = [];
             if (contacts) {
                 $.each(contacts.Persons, function (index, personData) {
-                    if (validateEmail(personData.email)) {
+                    if (validateEmail(personData.email) || personData.mobilephone != '') {
```
[the remark this answers is not in the document]
Comment (andreasastrom) [reply GetAccept-v2/app.js]: done in the VBA

**Comment by justFredster** [comment]
@andreasastrom @j-tegen  Great to see things happening! :) Just wanted to remind you to create a release, add to the changelog and update the documentation where necessary.

**Pull request merged**